packages/visx-shape/src/shapes/Pie.tsx

```diff
@@ -33,6 +35,8 @@ export type PieProps<Datum> = {
   pieSortValues?: PiePathConfig<Datum>['sortValues'];
   /** Render function override which is passed the configured arc generator as input. */
   children?: (provided: ProvidedProps<Datum>) => React.ReactNode;
+  /** Optional accessor function to return the fill string value of a given arc **/
```

Comment:
Super nit but these are transformed into docs so adding the period :)

```suggestion
  /** Optional accessor function to return the fill string value of a given arc. **/
```